samples/v1.3/Scenarios/InputFormWithLabels.json

```diff
@@ -51,9 +51,9 @@
 						{
 							"type": "Input.Text",
 							"id": "myTel",
-							"label": "Phone Number (xxx-xxx-xxxx)",
-							"regex": "^[0-9]{3}-[0-9]{3}-[0-9]{4}$",
-							"errorMessage": "Invalid phone number. Use the specified format: 3 numbers, hyphen, 3 numbers, hyphen and 4 numbers",
+							"label": "Phone Number (xxx xxx xxxx)",
+							"regex": "^\\(?([0-9]{3})\\)?[-.\\s]?([0-9]{3})[-.\\s]?([0-9]{4})$",
```

Comment:
Is xxx.xxx.xxxx expected?

Reply:
what formats are we planning to support? for example:
123-456-7890
(123) 456-7890
123 456 7890
123.456.7890
+91 (123) 456-7890

do we plan to support stuff like this? if yes, then this regex is probably not enough?:)